JS/Board.js

```diff
@@ -6,9 +6,36 @@ class Board {
     this.completed = false
   }
 
+
+  /////this needs refactoring
   populateBoard() {
     // runs dom fn
-    //gets values from dataset
+    let clues = (Object.values(data)[2])
+    console.log('clues', clues)
```

Comment:
Meep meep, also leave out the `console.logs` -- those are useful for you, while you're developing, but just add clutter to the diff for your partner to read through.